packages/legacy/core/App/screens/Connection.tsx

```diff
@@ -124,6 +124,10 @@ const Connection: React.FC<ConnectionProps> = ({ navigation, route }) => {
   useEffect(() => {
     if (state.shouldShowDelayMessage && !state.notificationRecord) {
       AccessibilityInfo.announceForAccessibility(t('Connection.TakingTooLong'))
+      if (autoRedirectConnectionToHome) {
+        dispatch({ shouldShowDelayMessage: false, isVisible: false })
+        navigation.getParent()?.navigate(TabStacks.HomeStack, { screen: Screens.Home })
+      }
```

Comment:
Should we only announce that for accessibility if we're *not* auto-redirecting to home?

Reply:
good catch, yes that's right